### ISSUE TITLE
[a11y]: Provide a single mouse click/tap way to resize a TextArea

### ISSUE THREAD

**Issue by philljenkins** [carbon-design-system/carbon#19902]
### Package

carbon-components-react

### Browser

Chrome

### Operating System

MacOS

### Package version

@carbon/react@1.86.0

### React version

@carbon/react@1.86.0

### Automated testing tool and ruleset

Checker - latest rule set

### Assistive technology

Single mouse click/tap 

### Description

TextArea components needs to meet [2.5.7 Dragging Movements](https://www.ibm.com/able/requirements/requirements/?version=v7_3#2_5_7).

The [Carbon react TextArea](https://react.carbondesignsystem.com/?path=/docs/components-textarea--overview) implements auto-show scroll-bar, but not auto-resize, is keyboard accessible, but does not meet 2.5.7 to allow single clicks/taps to resize the area (without dragging).

### WCAG 2.1 Violation

WCAG 2.2 TextArea components needs to meet [2.5.7 Dragging Movements](https://www.ibm.com/able/requirements/requirements/?version=v7_3#2_5_7).

### Reproduction/example

Single click on TextArea's resize handle - nothing happens. 

### Steps to reproduce

Here are a few alternatives to consider:
- **_Keyboard Control_**: this alternative DOES NOT MEET 2.5.7, but is required for 2.1.1 Keyboard.  So a keyboard alternative is also required but not sufficient.
- **_Context Menu_**: Implement a context menu for the text-area that includes options for resizing (e.g., "increase height," "decrease width").
- **_Input Fields_**: Provide numeric input fields (width and height) next to the text-area where users can enter desired dimensions directly.
- **_Buttons_**: Include dedicated "plus" and "minus" buttons to incrementally adjust the text-area's width or height.

### Suggested Severity

Severity 2 = User cannot complete task, and/or no workaround within the user experience of a given component.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by philljenkins** [comment]
**_Auto-resize_**: this is a partial solution to automatically increase the height when typing input but will not automatically show scroll-bars. it simply automatically increases the area to match and not show scroll-bars. So how will the user reduce the text-area without dragging? In my opinion a critical reason the design wants resizable text-area input is to allow the user (not the dev) to decide the best size and when to show or remove scroll-bars.

**Comment by philljenkins** [comment]
while the resizing functionality of a default HTML native text-area can be convenient for many users, it's not inherently accessible for everyone. Implementing an alternative resizing method alongside the default dragging behavior is necessary to ensure compliance with WCAG [2.5.7.](https://www.ibm.com/able/requirements/requirements/?version=v7_3#2_5_7)

**Comment by mbgower** [comment]
This is simply a textarea styled by Carbon. The functionality remains that of the default html component and I therefore believe this falls under the exception language:
>  the functionality is determined by the [user agent](https://www.w3.org/WAI/WCAG22/Understanding/dragging-movements.html#dfn-user-agent) and not modified by the author.

**Comment by mbgower** [comment]
If Carbon wishes to try to override the HTML behaviour, they are welcome to, but it is not needed to meet WCAG. I recommend instead the issue be opened against HTML and/or the user agents.

**Comment by philljenkins** [comment]
I open an issue with the ARIA APG - https://github.com/w3c/aria-practices/issues/681

**Comment by heloiselui** [comment]
We are closing this issue fo now. Since the behavior is controlled by the user agent, we’ll wait to see what comes out of the ARIA APG issue (w3c/aria-practices#681). If guidance changes or someone wants to propose a compliant pattern, we are happy to revisit in a new issue.
